conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -12,5 +12,5 @@
 # Add all the application routes to the prod.routes file
 ->          /                      prod.Routes
 
-GET         /update-and-submit-income-tax-return/self-employment/test-clear-all-data        controllers.testonly.TestManagerController.isTestEnabled()
-POST        /update-and-submit-income-tax-return/self-employment/test-clear-all-data        controllers.testonly.TestManagerController.clearAllData()
\ No newline at end of file
+GET         /update-and-submit-income-tax-return/self-employment/test-only/test-clear-all-data        controllers.testonly.TestManagerController.isTestEnabled()
```

Comment:
yes, good call to change it. You need to also update income-tax-submission-journey-tests project which use this endpoint in run_self_employment_tests.sh